GalaxyZooTouchTable/GalaxyZooTouchTable/ViewModels/ClassificationSummaryViewModel.cs

```diff
@@ -43,12 +45,25 @@ public void ProcessNewClassification(string subjectLocation, ClassificationCount
         {
             int index = random.Next(SummaryStrings.Count);
             ClassificationSummary = new ClassificationSummary(subjectLocation, counts, currentAnswers, selectedAnswer, SummaryStrings[index]);
+            TemporaryDisableButtons();
+        }
+
+        async void TemporaryDisableButtons()
+        {
+            ButtonsDisabled = true;
+            await Task.Delay(TimeSpan.FromSeconds(1));
+            ButtonsDisabled = false;
         }
 
         void LoadCommands()
         {
-            RandomGalaxy = new CustomCommand(OnRandomGalaxy);
-            ChooseAnotherGalaxy = new CustomCommand(OnChooseAnotherGalaxy);
+            RandomGalaxy = new CustomCommand(OnRandomGalaxy, CanBeginNewClassification);
+            ChooseAnotherGalaxy = new CustomCommand(OnChooseAnotherGalaxy, CanBeginNewClassification);
+        }
+
+        private bool CanBeginNewClassification(object obj)
+        {
+            return !ButtonsDisabled;
```

Comment:
I'm a big fan of using clear naming the way you have done here rather than sticking the inverse of ButtonsDisabled directly into the CustomCommand constructor 🥇